<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MXButtonConf</class>
 <widget class="QDialog" name="MXButtonConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/icons/logo.png</normaloff>:/icons/icons/logo.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbWeb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>501</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Mosaic Web Toolbar </string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="cbNoImages">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No Images</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbBack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAdd2hot">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add2Hot</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbNews">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>News</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbHome">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbReload">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbForward">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbCurHot">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Hotliist</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMai">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>E-Mail</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbStop">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbMosaic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Mosaic Toolbar</string>
   </property>
   <widget class="QCheckBox" name="cbOpenFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Open FIle</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSaveFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbCopy">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbFind">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbManager">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Manager Views</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbPrint">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbPrintPreview">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Print Preview</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbHelp">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSearchCache">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Cache</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbPaste">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MXButtonConf</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MXButtonConf</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
